sql/logic_test.go

```diff
@@ -72,6 +74,9 @@ type logicStatement struct {
 type logicQuery struct {
 	pos             string
 	sql             string
+	colNames        bool
```

Comment:
Well, each query has its own configuration so I'm not sure what we get with the separation.